example/static_images.js

```diff
@@ -88,14 +88,23 @@ $(function() {
                 length: 10
             },
             decoder : {
-                readers : ["code_128_reader"]
+                readers : ["code_128_reader"],
+                drawScanline: true,
```

Comment:
Oh doh, sorry, I should've cleaned this file back up more - I was fiddling with things trying to learn what did and didn't work and when/how.